### PR TITLE
Add location specific course info pages

### LINKED ISSUE #406: Landing page London/Nazareth links go to Github
the two links in the middle of the page, go to Github repos which is -in our opinion- bad UX. 
We propose very simple information pages for each, e.g at /programme/course-information/location

### PR DESCRIPTION
**Pull request by mattlub**
fixes #406 
related #387 

Created new pages:
/programme/course-information/nazareth/
/programme/course-information/london/

Changed all the links on the site to go these instead of the README's below.
https://github.com/foundersandcoders/info/blob/master/london.md
https://github.com/foundersandcoders/info/blob/master/nazareth.md

I left the content the same as the README's but they should be updated if they are major links from the front page of the site.

screen shot 2017-03-05 at 19 17 39

### CODE REVIEW

File: _harp/programme/course-information/nazareth/index.ejs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="section-copy">
```
Comment (des-des) [inline _harp/programme/course-information/nazareth/index.ejs]: Okay I see we have the same code repeated twice, with the only small change. The idea of building templates is to not repeat ourselves like this? Can you find a way to refactor out this repetition? Happy to lend a hand if you need help with this!

**Comment by rebeccaradding** [comment]
@mattlub might be worth incorporating this change too:  foundersandcoders/info#6

**Comment by des-des** [comment]
@rebeccaradding apparently that is being depreciated..

**Comment by mattlub** [comment]
cool. I've restructured the files. @rebeccaradding I'll make a different issue for that.

File: _harp/programme/course-information/index.ejs [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <div class="section-copy">
 	<div class="container">
-		<%- partial("_course-information.md") %>
+    <%- partial("_course-information.md") %>
```
Comment (des-des) [inline _harp/programme/course-information/index.ejs]: Indentation

File: _harp/programme/course-information/london.ejs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="section-copy">
+	<div class="container">
+    <%- partial("_london.md") %>
```
Comment (des-des) [inline _harp/programme/course-information/london.ejs]: Indentation

File: _harp/programme/course-information/nazareth.ejs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<div class="section-copy">
+	<div class="container">
+    <%- partial("_nazareth.md") %>
```
Comment (des-des) [inline _harp/programme/course-information/nazareth.ejs]: Indentation

**Comment by mattlub** [comment]
fixed! @des-des

**Pull request merged**